testing/microlith-test/integration/prometheus_alert_overrides/prometheus_alert_overrides.bats

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bats
+
+# Copyright 2021 Couchbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file  except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the  License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load "$BATS_SUPPORT_ROOT/load.bash"
+load "$BATS_ASSERT_ROOT/load.bash"
+load "$BATS_FILE_ROOT/load.bash"
+load "../helpers"
+
+export CMOS_IMAGE=${CMOS_IMAGE:-couchbase/observability-stack:v1}
```

Comment:
These should not be required, we do have a COS_IMAGE variable but anything we need should be set at a top-level. I'd rather not bury requirements away.

Reply:
If we set CMOS_IMAGE outside this file, then we wouldn't be able to run individual tests directly without setting it manually (i.e. couldn't do `bats prometheus_alert_overrides.bats` without explicitly setting `CMOS_IMAGE=...`) - is this intended?

Reply:
We should be setting it at a top-level (it is currently) I think. Otherwise we run into weird stuff were things are set differently or with typos, etc. If we want to run a specific test then we have to set it, and you can have a pre-reqs test case to fail early.

Reply:
I don't see it getting set anywhere top-level (not on this branch at least):

![image](https://user-images.githubusercontent.com/2904440/135832842-a70f6c31-9ab6-4784-a65b-10ef10a9eb36.png)

Reply:
Yeah it's currently COS_IMAGE in the container, I'm renaming that for the CICD purposes.
Basically we should have pre-reqs test that just verifies any env vars first I think then we can fail the rest.

Reply:
I'm setting all these now in the testing/native/run.sh script